mobile-app/app/screens/AppNavigator/screens/Portfolio/screens/SendScreen.tsx

```diff
@@ -139,6 +143,9 @@ export function SendScreen({ route, navigation }: Props): JSX.Element {
     let infoText;
     let themedProps;
     let status = TransactionCardStatus.Default;
+    const isEvmAddress =
+      getAddressCategory(getValues("address"), networkName) ===
```

Comment:
Didnt know `getAddressCategory` function exists, been using `getAddressType` all this time 🤔

Reply:
Oh no, I just renamed it in the import because there's already a getAddressType function - to not confuse devs of the existing "address type" context" in LW